samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/MainApplication.kt

```diff
@@ -19,31 +15,35 @@ import kotlinx.coroutines.runBlocking
 class MainApplication : Application() {
 
     @Inject
-    lateinit var dataStore: DataStore<Preferences>
+    lateinit var preferences: PreferenceRepository
+
     override fun onCreate() {
         super.onCreate()
-        val (trackingApiUrl, siteId, apiKey) = runBlocking {
-            return@runBlocking try {
-                val data = dataStore.data.first()
-                listOf(
-                    data[TRACK_API_URL_KEY],
-                    data[SITE_ID],
-                    data[API_KEY]
-                )
-            } catch (e: Exception) {
-                listOf(null, BuildConfig.SITE_ID, BuildConfig.API_KEY)
-            }
+        val configuration = runBlocking {
+            preferences.getConfiguration().first()
```

Comment:
Does it guarantee it will return some value? With fresh install as well? 🤔

Reply:
Yes it initializes it with `defaults`